metaflow/plugins/kfp/constants.py

```diff
@@ -3,9 +3,11 @@
 # Defaults for running MF on KFP
 # TODO: remove this when we're able to automatically package code and use it on KFP. We currently need the user supplied code URL (or this default URL) in order to download the code that is to be run on KFP
 DEFAULT_FLOW_CODE_URL = 'https://gist.githubusercontent.com/mon95/98d9d88571a0307a53b637d841295702/raw/df8bc1fa363f629271cfc1d34f3c2f7ca0b2e2cf/helloworld.py'
+DEFAULT_DOWNLOADED_FLOW_FILENAME = 'downloaded_flow.py'
+
 DEFAULT_KFP_YAML_OUTPUT_PATH = 'kfp_pipeline.yaml'
 DEFAULT_RUN_NAME = 'run_mf_on_kfp'
 DEFAULT_EXPERIMENT_NAME = 'mf-on-kfp-experiments'
 
 # TODO: This should (probably) be moved outside
-RUN_LINK_PREFIX = "https://kubeflow.corp.zillow-analytics-dev.zg-int.net/pipeline/#/runs/details/"
\ No newline at end of file
+RUN_LINK_PREFIX = "https://kubeflow.corp.zillow-analytics-dev.zg-int.net/pipeline/#/runs/details/"
```

Comment:
Let's move this to `metaflow_config.py` using `from_conf`.  You can comment what the path would be for users to know what it should point to.

The name would be `KFP_RUN_URL_PREFIX`.

Reply:
tip: I searched for "zillow" and this is the only variable/constant remaining I believe should move to `metaflow_config.py`

Reply:
Done now! Thanks